### DISCUSSION TITLE
Is there a way to access llava through the api?

### DISCUSSION THREAD

**Discussion by samsal** [oobabooga/text-generation-webui#1582] [Q&A]
It would be really nice to have the possibility to make POST request to the API in which there is a picture is sent in base64 format and the prompt that goes along with it.

**Comment by Wojtab** [comment]
there is, you just send base64 jpeg, for an example see new README.md in https://github.com/oobabooga/text-generation-webui/pull/1664

**Reply by xts-bit** [reply]
@Wojtab How to do that? any API NOW?

**Reply by lgiavedoni** [reply]
here: https://github.com/oobabooga/text-generation-webui/pull/1664/commits/38ab214a416b2dbb6bcba4d318bfc847fbb4da36
